### ISSUE TITLE
linter doesn't do anything

### ISSUE THREAD

**Issue by warpdesign** [AtomLinter/Linter#616]
I have installed linter and both linter-json and linter-jshint packages but no matter what file I open, it shows 0 errors, "currentFile; 0, project: 0".

Even if I type some broken JavaScript, it shows "No errors".

I tried to restart Atom, still the same problem. I was using version 0.211.0. I updated to version 1.0 and still get no error reported.

**Comment by warpdesign** [comment]
I'm on Windows btw.

**Comment by steelbrain** [comment]
Are you using the latest version of `linter`?
They are working for me on linux, Can anyone else on windows confirm that behavior?
Also Joe Fitzgerald uses windows I think but hasn't reported this error yet. 
/cc @AtomLinter/linter

**Comment by warpdesign** [comment]
Yes: I'm using version 1.0.4 which appears to be the latest version.

**Comment by warpdesign** [comment]
After some investigation, I disabled every third party packages I was using and kept only linter/linter-jshint: same problem.

Then I disabled the core package Autocomplete-plus, restarted Atom and.. it worked!

**Comment by steelbrain** [comment]
Glad to know that you found the problem, the `autocomplete-plus` package is actually a base package, did you try disabling some of it's providers to see if it comes from them.

**Comment by passcod** [comment]
+1 Can reproduce

**Comment by steelbrain** [comment]
@passcod: Can you please tell us the linters you're using?

**Comment by passcod** [comment]
I edited with the list of atom packages I have, that should answer that. I've currently disabled all autocomplete packages and am going through enabling them one by one to pinpoint the issue.

**Comment by steelbrain** [comment]
@passcod: Which of those linters have you tested and found to be non-working?

**Comment by passcod** [comment]
jscs, jshint, php.

**Comment by k2b6s9j** [comment]
Then we should put a priority on transitioning those linters to the new api.

**Comment by steelbrain** [comment]
I can confirm that `jshint` works, what is your Atom version by the way?

**Comment by passcod** [comment]
As mentioned above, 1.0.0.

**Comment by steelbrain** [comment]
@k2b6s9j sounds like he's using an older version of atom, I've confirmed that one of them works, the others too, probably.

**Comment by steelbrain** [comment]
@passcod Can you do
```
mv ~/.atom ~/.atom-bkp
apm install linter
apm install linter-jshint
```
and tell us the results?

**Comment by passcod** [comment]
Yeah, that works, but then again I already knew the linters weren't the cause... as mentioned, *disabling autocomplete fixed it*.

From OP:
> Then I disabled the core package Autocomplete-plus, restarted Atom and.. it worked!

Then I did this:
> I've currently disabled all autocomplete packages and am going through enabling them one by one to pinpoint the issue.

...and after re-enabling all my autocomplete plugins, I can't reproduce anymore.

Methinks it's an upgrade issue, when going from an older Atom to 1.0.0 with linters + the built-in autocomplete, they... conflict somehow?

A coworker's setup has the same issue, I may be able to get them to run a few diagnostic commands if you want another shot at this, but my experience is that once you disable then re-enable the autocomplete plugins the issue disappears, so I guess it's going to be hard to track.

**Comment by steelbrain** [comment]
Closing as not really a bug in Linter. will continue discussing.
